plugins/catalog-import/src/components/ImportStepper/defaults.tsx

```diff
@@ -217,14 +230,30 @@ export function defaultGenerateStepper(
                     </Box>
 
                     <TextField
-                      name="componentName"
-                      label="Name of the created component"
-                      placeholder="my-component"
+                      name="entityName"
+                      label="Name of the created entity"
+                      placeholder="my-entity"
                       margin="normal"
                       variant="outlined"
                       fullWidth
                       inputRef={register({ required: true })}
-                      error={Boolean(errors.componentName)}
+                      error={Boolean(errors.entityName)}
+                      required
+                    />
+
+                    <AutocompleteTextField
+                      name="type"
+                      control={control}
+                      errors={errors}
+                      freeSolo={false}
```

Comment:
I thought otherwise when coding it but I think you're right. What about just suggesting the list of "standard" types while allowing the user to freely type something else? Eventually, there might be away for users or admins to configure their own list of supported types?

Reply:
In summary if you agree, I'll remove the freeSolo option